src/components/Calendar.scss

```diff
@@ -1,127 +1,141 @@
+  padding-bottom: $su-small;
+  border-left: $border-width-small solid $ts-grey;
+  border-bottom: $border-width-small solid $ts-grey;
+}
+.Calendar-cell--currentDay {
+  background: $cu-primary--bg;
+  p, abbr {
+    color: $cu-primary;
+    font-weight: $tu-bold-fontWeight;
   }
-  & > [data-ui-component="next-month"] {
-    background: $cu-background--light;
-    span, abbr {
-      opacity: 0.6;
-    }
+}
+.Calendar-cell--nextMonth {
+  background: $cu-background--light;
+  p, abbr {
+    opacity: 0.6;
   }
 }
 
-// responsive/widget
+// * 3. Widget Version
 
 .Calendar-widget--exampleWrapper {
   width: 300px;
 }
 
 .Calendar.Calendar-widget {
-  .Calendar-secondary-heading {
-    text-align: center;
-    .Button--paginate {
-      float: left;
-      &:last-child {
-        float: right;
-      }
-    }
-
-  }
-  .Calendar-row[data-ui-component="week-heading"] li {
-    display: block;
-  }
-  .Calendar-row[data-ui-component="week"] {
-    & > [class*="Calendar-cell"] {
-      height: 40px;
-      text-align: center;
-    }
-    & > [data-ui-component="next-month"] {
+  .Calendar-cell--nextMonth {
     background: $ts-white;
-    span, abbr {
+    p, abbr {
       display: none;
     }
   }
-  }
   .Calendar-row {
     box-sizing: border-box;
     display: flex;
-    -webkit-box-orient: horizontal;
-    -webkit-box-direction: normal;
-    flex-flow: row wrap;
   }
   .Calendar-cell {
-    -webkit-box-flex: 1;
     flex: 1 1 0%;
+    min-height: 0;
+    height: auto;
     box-sizing: inherit;
     min-width: 0;
-    padding: $su-small + 2;
+    text-align: center;
+    padding: $su-small 0;
+    &.Calendar-cell--dayHeading span {
+      display: none;
+    }
+    &.Calendar-cell--dayHeading[title="Monday"]:after {
+      content: 'Mo';
+    }
+    &.Calendar-cell--dayHeading[title="Tuesday"]:after {
+      content: 'Tu';
+    }
+    &.Calendar-cell--dayHeading[title="Wednesday"]:after {
+      content: 'We';
+    }
+    &.Calendar-cell--dayHeading[title="Thursday"]:after {
+      content: 'Th';
+    }
+    &.Calendar-cell--dayHeading[title="Friday"]:after {
+      content: 'Fr';
+    }
+    &.Calendar-cell--dayHeading[title="Saturday"]:after {
+      content: 'Sa';
+    }
+    &.Calendar-cell--dayHeading[title="Sunday"]:after {
+      content: 'Su';
+    }
     abbr {
       display: none;
     }
   }
 }
 
-@media only screen and (min-width: 768px) {
-  .Calendar-row[data-ui-component="week-heading"] {
-    text-align: center;
-  }
-  .Calendar-row[data-ui-component="week"] > [class*="Calendar-cell"] {
-    height: 150px;
-    padding-bottom: 0;
-  }
-  .Calendar-row[data-ui-component="week-heading"] li {
-    display: block;
-  }
+// * 4. Responsive considerations
+
+@media only screen and (max-width: $breakpoint-sm) {
   .Calendar-row {
-    box-sizing: border-box;
-    display: flex;
-    -webkit-box-orient: horizontal;
-    -webkit-box-direction: normal;
-    flex-flow: row wrap;
+    display: block;
+    &.Calendar-row--weekHeading {
```

Comment:
Same chaining question as above... the modifier class _should_ be all you need here.